jekyll/_cci2/concurrency.adoc

```diff
@@ -0,0 +1,114 @@
+---
+version:
+- Cloud
+- Server 3
+- Server 2
+---
+= Concurrency
+:page-layout: classic-docs
+:page-liquid:
+:page-description: This documentation summarizes concurrency, and suggests ways to use concurrency to your advantage.
+:icons: font
+:toc: macro
+:toc-title:
+
+This documentation summarizes concurrency, and suggests ways to use concurrency to your advantage. 
+
+In CircleCI, **concurrency** refers to utilizing multiple containers to run multiple builds at the same time. This is different from CircleCI **parallelism**, which is test-splitting across multiple containers. If you would like information on parallelism, visit the <<parallelism-faster-jobs/#,Running Tests in Parallel>> page.
+
+Every plan with CircleCI includes the ability to concurrently run jobs. However, the number of jobs you can run concurrently differs between plans. Please view the https://circleci.com/pricing/[pricing page] to see up to date information on the number of concurrent job runs included in your plan.
+
+toc::[]
+
+== Concurrency
+Concurrency allows multiple jobs to be run at the same time in a workflow through the use of multiple containers. To keep the system stable for all CircleCI customers, we implement different soft concurrency limits on each of the <<configuration-reference/#resourceclass,resource classes>> for different executors. 
```

Comment:
I think it would be good to allude to the bigger picture - org-level concurrency here (slack conversation about this: https://circleci.slack.com/archives/C48NG2XFW/p1576777817012200?thread_ts=1576755015.008200&cid=C48NG2XFW). Also would you mind updating the entry for concurrency on the concepts page to match what you have here? I notice it's almost the same but uses "build" which I think we wanted to avoid.